rudder-lang/src/technique/outcome.rs

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// SPDX-FileCopyrightText: 2019-2020 Normation SAS
+
+use crate::error::*;
+use std::{fmt, str::FromStr};
+use strum::{EnumIter, IntoEnumIterator};
+
+#[derive(EnumIter, PartialEq, Debug)]
+pub enum ConditionOutcome {
```

Comment:
Literal outcomes as used in return statements should only be Repaired, Kept or Error